### ISSUE TITLE
Video Context

### ISSUE THREAD

**Issue by robertSt7** [pimcore/studio-ui-bundle#492]
see https://github.com/pimcore/studio-ui-bundle/pull/457#discussion_r1713276725

**Comment by markus-moser** [comment]
@vin0401 Is this topic still relevant?

**Comment by vin0401** [comment]
yep, it's still a improvement I would like to make. But nothing really urgent, I guess.
